commissioning.tex

```diff
@@ -1,27 +1,48 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.
```

Comment:
Formally only defined for LSSTCam.

Reply:
I think it would assist users' comprehension if we just started saying "LSSTCam First Photon", throughout, then.  In particular that will help users understand that the 1.5-month period you suggest below doesn't include the time during which we'll be tuning up the AOS on ComCam.

Reply:
I originally understood "First Photon" to refer to the first photons through the optical system, irrespective of which camera they fell on. But I'm not sure now. If "First Photon" is to refer only to the first photons through the optical system and imaged by LSSTCam then we need another milestone ComCam on sky, as  a lot of the work that would be done after the original "LSSTCam First Photon" now gets done with ComCam, resulting in the shorter time between FP and SFL (as Gregory points out)

Reply:
All definitions now updated